### DISCUSSION TITLE
请问当前最新版 v4.0.52，怎么开启 build & watch 这个功能

### DISCUSSION THREAD

**Discussion by nisnaker** [umijs/umi#10569] [Q&A] [answered]
v4.0.52 的 changelog 第一条是 ` 新增 build 支持 watch by @PeachScript `
但是官网文档里没看到怎么开启，umi help dev / umi help build 也没有说明，
我乱试的 `umi build --watch` `umi dev --build` 也都是build一次就退出了，请问这个功能怎么开启，因为在开发chrome插件，dev的产物在chrome里会报错，build 的正常

**Comment by fz6m** [comment, marked as answer]
这个功能是给 father 用的，可以看看 father umd 构建能不能满足你的需求。

**Reply by nisnaker** [reply]
好的，非常感谢

**Comment by nisnaker** [comment]
自己回一下，关掉 mfsu 之后 dev 也正常了
